data/json/items/armor/torso_clothes.json

```
@@ -653,7 +653,7 @@
     "repairs_like": "jeans",
     "type": "ARMOR",
     "name": { "str": "santa dress", "str_pl": "santa dresses" },
-    "description": "A red sleeved dress with white faux fur as trim.  For when you want to be Mrs.  Claus on Christmas, or perhaps Santa going clubbing.",
+    "description": "A red sleeved dress with white faux fur as trim.  For when you want to be Mrs. Claus on Christmas, or perhaps Santa going clubbing.",
```

```suggestion
    "description": {
      "//NOLINT(cata-text-style)": "The dot after Mrs is not a period",
      "str": "A red sleeved dress with white faux fur as trim.  For when you want to be Mrs. Claus on Christmas, or perhaps Santa going clubbing."
    },
```
This suppresses the warning.